Report polishing

aya-codes opened the pull request.

MaheshNanavare committed:
Update figure names in ReadMe file

Add YouTube demo video link,
update figure names,
correct Github to GitHub,
correct P5.js to p5.js,
update repeating alt text for some images

MaheshNanavare committed:
Delete docs/.eslintrc.json

MaheshNanavare committed:
Delete docs/.prettierrc

MaheshNanavare committed:
Update use case diagram

Replace mobile with touch screen device

MaheshNanavare committed:
Delete documentation/other_docs/use_case.png

use_case.jpeg will be used instead of use_case.png as it is taking less space for same quality.

MaheshNanavare committed:
Add gifs from demo video

MaheshNanavare committed:
Rename Challenge_1.gif to Challenge_2.gif

MaheshNanavare committed:
Update challenge 2 gif

MaheshNanavare committed:
Delete documentation/other_docs/Aya's Tasks

MaheshNanavare committed:
Add GitHub Actions Usage Matrix

MaheshNanavare committed:
Add gifs from demo video to README file

- Basic_game_version
- Challenge_2
- User_Testing

MaheshNanavare committed:
Add Sustainability Calculation Indicator in README file

aya-codes committed:
Update README.md

added images to objects table, adjusted some other things

MaheshNanavare reviewed (commented):
Things done in the commit:
- Internal links updated in index
- Class names' formatting updated
- System Architecture content updated
- Figure numbers updated
- Use case specifications updated to table format
- Game entities images added and names updated

MaheshNanavare committed:
Update Bibliography sequence in README.md

- Update Bibliography sequence
- Add internal links to respective Bibliography point
- Center formula of SCI
- Correct location of titles Figure 24 and 25

MaheshNanavare reviewed (approved):
Everything is perfect except:
- Need to correct Bibliography internal links
